include/drivers/sensor.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_SENSOR_H
+#define DRIVERS_SENSOR_H
+
+#include <dm.h>
+#include <intrusive.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <util.h>
+
+#define SENSOR_OPS(dev) \
+	(&container_of((dev)->drv, struct sensor_driver, drv)->ops)
+
+struct sensor_info {
+	const char *const name;       /**< Name exported to SCPI. */
+	const uint8_t     offset;     /**< offset of register. */
```

Comment:
The name was fine. I understood exactly what it was for. The problem is that _it doesn't belong in this struct_; it belongs in _a driver-specific struct_. You should have nested structs in the driver, like in `axp803` (and `sunxi-ccu` etc.),

Reply:
done